tests/test_alloc.rs

```diff
@@ -5,15 +5,17 @@ use deku::prelude::*;
 static A: AllocCounterSystem = AllocCounterSystem;
 
 #[derive(Debug, PartialEq, DekuRead, DekuWrite)]
+#[deku(ctx = "is_le_bytes: bool")]
 struct NestedStruct {
+    #[deku(ctx = "is_le_bytes")]
     pub field_a: u8,
 }
 
 #[derive(Debug, PartialEq, DekuRead, DekuWrite)]
-#[deku(id_type = "u8")]
+#[deku(id_type = "u8", ctx = "is_le_bytes: bool")]
 enum NestedEnum {
     #[deku(id = "0x01")]
-    VarA(u8),
+    VarA(#[deku(ctx = "is_le_bytes")] u8),
```

Comment:
are these changes intended?

Reply:
Yes, they are. As I mentioned in [this comment](https://github.com/sharksforarms/deku/pull/61#issuecomment-661894039), a top-level endian will imply that all fields require a `endian ctx`. So I added `ctx = "is_le_bytes: bool"` for its subtype. But because the field level `endian` (for now) only accept "big" or "little"(we want to use variant here), I added `#[deku(ctx = "is_le_bytes")] u8`.